### PR TITLE
Add a few missing pieces of osp-13 feature branch

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

File: playbooks/files/rax-maas/plugins/memcached_status.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 
 VERSION_RE = re.compile('STAT version (\d+\.\d+\.\d+)(?![-+0-9\\.])')
-VERSIONS = ['1.4.14 (Ubuntu)', '1.4.15', '1.4.25 Ubuntu']
+VERSIONS = ['1.4.14 (Ubuntu)', '1.4.15', '1.4.25 Ubuntu', '1.4.39']
```
Comment (npawelek) [inline playbooks/files/rax-maas/plugins/memcached_status.py]: I wonder if this is still valid? #574 has these options removed to support Rocky, so we might be able to do the same with RPCR work.

Comment (tonytan4ever) [reply playbooks/files/rax-maas/plugins/memcached_status.py]: Yes,  we'll test it out on an osp13 instance and see if it works. For now, we'll keep just the as it is.

**Comment by tonytan4ever** [comment]
recheck_container_docs_tox-test
recheck_ container_pep8_tox
recheck_ xenial_ceph_deploy
recheck_ xenial_newton_deploy

**Comment by tonytan4ever** [comment]
recheck_container_pep8_tox-test
recheck_ xenial_ceph_deploy
recheck_xenial_newton_deploy

**Comment by tonytan4ever** [comment]
recheck_ xenial_ceph_deploy

**Comment by tonytan4ever** [comment]
recheck_xenial_queens_deploy
recheck_xenial_pike_deploy

**Comment by tonytan4ever** [comment]
recheck_trusty_newton_deploy
recheck_trusty_mitaka_deploy
recheck_trusty_liberty_deploy

**Comment by tonytan4ever** [comment]
recheck_xenial_newton_deploy

**Pull request merged**